### ISSUE TITLE
Functions host fails to create "leases" collection

### ISSUE THREAD

**Issue by adrianknight89** [Azure/azure-webjobs-sdk-extensions#577]
I have multiple `CosmosDBTriggers` set up with `LeaseCollectionName = "leases", CreateLeaseCollectionIfNotExists = true` attributes. A console app deletes and recreates all collections (it also deletes the `leases` collection) on a regular basis. When a function is run, the associated trigger is supposed to create the `leases` collection if it doesn't exist. Unfortunately, sometimes this collection is not created.

**Comment by adrianknight89** [comment]
@kashimiz I will provide the requested information next time the issue happens. I added a lease collection prefix to every trigger since they were sharing the same collection and did not observe the issue since then. However, I deleted & created the collections only once after this change. If I observe the issue again, I will provide the information.

**Comment by v-anvari** [comment]
Hi @adrianknight89, We are proceeding to close the ticket since there were no visible errors. Please feel free to open a new issue in case of any further errors occur
